walletconsole/lib/CommandExecutor.cpp

```diff
@@ -84,7 +84,7 @@ void CommandExecutor::help() const {
     _out << "> coin bitcoin <Enter> newKey <Enter> addrPri # <Enter> fileW btcaddr.txt #" << endl;
     _out << "> newKey <Enter> pubPri # <Enter> addrPub #" << endl;
     _out << "> coin algo <Enter> addr LCSUSBOLNVT6BND6DWWGM4DLVUYJN3PGBT4T7LTCMDMKS7TR7FZAOHOVPE" << endl;
-    _out << "> coin btc <Enter> dumpDP <Enter> pridb <Enter> priDP m/84'/0'/0'/0/1 <Enter> pubPri # <Enter> addrPub # <Enter> addr #" << endl;
+    _out << "> coin btc <Enter> dumpDP <Enter> priDP <Enter> priDP m/84'/0'/0'/0/1 <Enter> pubPri # <Enter> addrPub # <Enter> addr #" << endl;
```

Comment:
can we make more intuitive names?

Reply:
I have typing too much.  DerivationPath is shortend to DP.  Pub and Pri are quite intuitive, I suppose.

Reply:
not intuitive, if you build this for internal use it might work, but not for external usage as most people not very familiar with this terminology

Reply:
This is dev tool and for hard core users